java/java.lsp.server/vscode/package.json

```diff
@@ -121,7 +121,7 @@
 				},
 				"netbeans.revealActiveInProjects": {
 					"type": "boolean",
-					"default": true,
+					"default": false,
```

Comment:
NBLS 13.0.301 release is over. Is there a plan to change the default to `true` for NetBeans 14 release, @sdedic?